pkg/cmsis-nn/Makefile.include

```diff
@@ -0,0 +1,6 @@
+INCLUDES += -I$(PKGDIRBASE)/cmsis-nn/CMSIS/NN/Include
+
+# Required for some basic math functions
+INCLUDES += -I$(PKGDIRBASE)/cmsis-nn/CMSIS/DSP/Include
+
+CFLAGS += -Wno-sign-compare
```

Comment:
already in `Makefile`

Reply:
I kept this one and removed the other: the warning occurs in a header file and is not silenced from the package `Makefile`. So I have to silent it globally unfortunately.